index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dan's portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section id="skeleton">
+      <header>
+        <h1 id="welcome">Welcome</h1>
+        <a href="#" id="menu">
+          <img src="images/menu.svg" alt="Drop down menu" />
+        </a>
+      </header>
+      <div id="page">
+        <div id="intro">
+          <p id="denis">Hey there. I’m Denis</p>
```

Comment:
- [ ] As the last reviewer requested Please wrap the main title in the h1 tag since the p element represents a paragraph.